src/test/scala/chisel3/iotesters/ZeroWidthIOSpec.scala

```diff
@@ -5,7 +5,7 @@ package chisel3.iotesters
 import chisel3._
 import org.scalatest._
 
-class MyZeroWidthDut extends chisel3.experimental.MultiIOModule {
+class MyZeroWidthDut extends chisel3.MultiIOModule {
```

Comment:
Nit: You probably don't need the chisel3. in front of MultiIOModule with the wild card chisel3._ above